testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -153,6 +154,8 @@
 
     @Before
     public void beforeAbstractKeycloakTest() throws Exception {
+        TestClassProvider.getInstance();
```

Comment:
Perhaps `AuthServerTestEnricher.initializeSuiteContext` or other arquillian `@BeforeSuite` event handler would be a better fit for this.

Reply:
thanks, was looking for something like that